judge/widgets/mixins.py

```diff
@@ -34,7 +34,7 @@ class CompressorWidgetMixin(object):
     except ImportError:
         pass
     else:
-        if getattr(settings, 'COMPRESS_ENABLED', not getattr(settings, 'DEBUG', False)):
```

Comment:
Semantics actually changed. Before, if you override `DEBUG`, and `COMPRESS_ENABLED` is not set, then `DEBUG` will be used. Now you have to override `COMPERSS_ENABLED` explicitly.